control-plane/plugin/src/resources/pool.rs

```diff
@@ -68,6 +70,10 @@ pub struct GetPoolsArgs {
     #[clap(long)]
     node_id: Option<NodeId>,
 
+    /// Gets Pools for the given volume.
+    #[clap(long)]
```

Comment:
```suggestion
    #[clap(long = "volume")]
```

Just a thought, would naming the args without the id, be a better ux?
> rest-pugin get pools --volume xxxx --node xxx

Reply:
```
[nix-shell:~/code/rust/mayastor-extensions]$ ./target/debug/kubectl-mayastor get volumes
 ID                                    REPLICAS  TARGET-NODE    ACCESSIBILITY  STATUS  SIZE  THIN-PROVISIONED  ALLOCATED  SNAPSHOTS  SOURCE
 bdc2941c-a79f-4134-b1d5-7e5c911a1b1d  1         node-1-184487  nvmf           Online  1GiB  false             1GiB       0          <none>
 ``` 
The output of `get volumes` has ID as header for volume , so IMHO `volume-id` is ok.

Reply:
It's generally the type followed by the id for get using id in kubectl, so maybe let's follow the same?

Reply:
What do you mean by that @Abhinandan-Purkait, can you provide an example?

Reply:
I meant we should use --volume instead of volume_id

Reply:
Yeah I also prefer --volume and --node, it reads better. If this is also a general approach then IMHO we should follow it.

Reply:
done